cli/src/main/java/dev/starfix/Editor.java

```diff
@@ -0,0 +1,40 @@
+package dev.starfix;
+
+import java.nio.file.Path;
+import java.io.IOException;
+import static dev.starfix.Starfix.runCommand;
+
+
+public class Editor{
+
+    public static void launch_editor(Path directory, String ide, String path, String filePath) throws IOException, InterruptedException {
```

Comment:
moving static method into a new class doen't change much - just adds another file but keep same coupling. 

should be a non-static method and have class per unique enough cases of IDE's